frontend/src/pages/Workflows/WorkflowDialogContent.js

```diff
@@ -174,11 +176,14 @@ const WorkflowDialogContent = (props) => {
           storeWorkflowAmount={storeWorkflowAmount}
           storeWorkflowAmountType={storeWorkflowAmountType}
           storeWorkflowCurrency={storeWorkflowCurrency}
+          storeWorkflowFundingOrga={storeWorkflowFundingOrga}
           workflowAmount={workflowToAdd.amount}
           storeWorkflowExchangeRate={storeWorkflowExchangeRate}
           exchangeRate={workflowToAdd.exchangeRate}
           workflowAmountType={workflowToAdd.amountType}
           workflowCurrency={workflowToAdd.currency}
+          fundingOrganization={workflowToAdd.fundingOrganization}
```

Comment:
@MartinJurcoGlina when you change the value of the dropdown you get an error from Mui 
<img width="502" alt="image" src="https://github.com/user-attachments/assets/95b2e727-0583-4a8c-9dc1-85fffcb2e1cb">
You can fix it multiple ways, e.g by always providing a default non-undefined value. E.g.

```suggestion
          fundingOrganization={workflowToAdd.fundingOrganization || ""}
```